front/src/components/article/textHtml.component.tsx

```diff
@@ -46,6 +51,11 @@ const TextHtml: FC<Props> = ({ html, offsetTotal }) => {
     content = content.replace("</oembed></figure>", "</iframe>");
     // Permet de corriger les liens des vidéos youtube
     content = content.replace("youtube.com/watch?v=", "youtube.com/embed/");
+    // Permet de corriger la largeur du contenu trop élevée avec des puces
+    content = content.replaceAll(
```

Comment:
Tu n'as pas de warning du type `replaceAll of undefined` comme j'avais pu avoir avec l'UUID ?

Reply:
Je n'ai pas de warning. Ca fonctionne sur l'émulateur mais pas sur le device